src/psi_environment/environment.py

```diff
@@ -13,34 +13,32 @@ class Environment:
     def __init__(
         self,
         random_seed: int = None,
-        agents: list[Car] = [],
+        agent_type: Type[Car] = None,
         ticks_per_second: int = 10,
+        n_bots: int = 10,
+        n_points: int = 3
     ):
         if random_seed is None:
             random_seed = random.randint(0, 2137)
         self._random_seed = random_seed
         np.random.seed(self._random_seed)
-        self._map = Map(self._random_seed)
+        self._map = Map(self._random_seed, n_bots, agent_type, n_points)
         self.data = EnvironmentAPI(self._map)
         self._game = Game(
             self._map, random_seed=random_seed, ticks_per_second=ticks_per_second
         )
         self._is_running = True
+        self.cost = 0
 
-    def step(self, action: int) -> tuple[Any, float, bool]:
-        """
-        Accepts action provided by agent when has to decide where to go. After that environment continues simulation
-        and returns another observation of the environment and cost it took by taking the action.
-        :param action: an integer (in range from 0 to 2) of direction where the agent wants to go.
-        :return: tuple of environment observation, cost of this action and bool stating if the agent has finished
-        """
-        # TODO: This requires a focus if only this will be returned.
-        #  Also I'm not sure about how observation will look like.
-        #  How will be direction handled so it will be consistent. I suggest something like a clock
-        #  but it will require some extra checks of edges.
-        #  And the implementation of it has not started
+    def step(self) -> bool:
         self._map.step()
-        return self._game.step(action)
+        self._game.step()
+        self.cost += 1
+        if self._map.is_game_over():
+            self._is_running = False
+            print("Game over!")
+            print(f"Cost: {self.cost}")
+        return self._is_running
```

Comment:
Dodałbym to, żeby zwracany również był aktualny koszt, żeby można było go odczytać z poziomu maina jak się zakończy symulacja